lib/rules/web/admin_console/4.6/monitoring_alerts.xyaml

```diff
@@ -284,4 +291,46 @@ hide_show_alert_graph_button:
     selector:
       xpath: //span[@class='pf-c-button__text']/parent::button
     op: click 
- 
\ No newline at end of file
+ 
+Disable_all_filters:
+  element:
+    selector:
+      xpath: (//button[text()='Clear all filters'])[2]
+    op: click
+clear_specifc_filter:
+  element:
+    selector: &specific_filter
+      xpath: //span[text()='<filter_category>']/..//button
+    op: click
+clear_name_filter_if_exists:
+  params:
+    filter_category: Name
+  action:
+    if_element:
+      selector:
+        <<: *specific_filter
+    ref: clear_specifc_filter
+list_alerts_by_filters_clear:
```

Comment:
the function name is not good

Reply:
any suggestion? here the filter work on alerts/silence/alert rule, all alerts related. I feel the name is ok

Reply:
ok